### PR TITLE
test: update tests to handle failures due to data change on PROD

### PR DESCRIPTION
**Pull request by ashwinimore**

### CODE REVIEW

File: e2e/components/action-bar/action-bar.async.po.js [hunk 1]
```diff
@@ -18,7 +18,15 @@ class ActionBarComponent {
...
+  
+  get deleteButton() {
+    return $(this.elements.delete);
+  }
 
+  get saveButton() {
+    return $(this.elements.save);
+  }
+  
   async closeMessages() {
```
Comment (andlar) [inline e2e/components/action-bar/action-bar.async.po.js]: I don't see either of these being used. I'm also not 100% sure these types of getters can be used in an `async` fashion, though I could be wrong

File: e2e/pages/surveillance/reporting/quarterly/quarterly.po.js [hunk 2]
```diff
@@ -154,7 +154,7 @@ class QuarterlyPage {
 
   async setSurvData(fields) {
     await (await $(this.elements.outcome)).selectByAttribute('value', fields.outcome);
-    await (await $(this.elements.processType)).selectByAttribute('value', fields.processType);
+    // await (await $(this.elements.processType)).selectByAttribute('value', fields.processType);
```
Comment (andlar) [inline e2e/pages/surveillance/reporting/quarterly/quarterly.po.js]: Probably should be removed, or uncommented out

File: e2e/pages/surveillance/reporting/quarterly/quarterly.spec.js [hunk 3]
```diff
@@ -64,7 +64,7 @@ describe('the quarterly surveillance reporting page', () => {
     it('can edit surveillance data of relevant listings under quarterly report', async () => {
       const surData = {
         outcome: '3',
-        processType: '1',
+        // processType: '3',
```
Comment (andlar) [inline e2e/pages/surveillance/reporting/quarterly/quarterly.spec.js]: Probably should be removed, or uncommented out

File: e2e/pages/surveillance/reporting/quarterly/quarterly.spec.js [hunk 4]
```diff
@@ -87,7 +87,7 @@ describe('the quarterly surveillance reporting page', () => {
       await browser.waitUntil(async () => (await quarterlyPage.progressBar).isDisplayed());
       await quarterlyPage.editSurveillanceData();
       await expect(await (await quarterlyPage.outcome).getValue()).toBe(surData.outcome);
-      await expect(await (await quarterlyPage.processType).getValue()).toBe(surData.processType);
+      // await expect(await (await quarterlyPage.processType).getValue()).toBe(surData.processType);
```
Comment (andlar) [inline e2e/pages/surveillance/reporting/quarterly/quarterly.spec.js]: Probably should be removed, or uncommented out

Comment (ashwinimore) [reply e2e/pages/surveillance/reporting/quarterly/quarterly.spec.js]: updates pushed to remove commented lines

**Review by andlar — APPROVED** [review]

**Review by tmy1313 — APPROVED** [review]

**Pull request merged**